src/main/java/com/hpe/simulap/protocol/sip/config/SipNodeElement.java

```diff
@@ -81,7 +82,7 @@ public void testStarted() {
 		_logger.info("testStarted()" + stackLog);
 
 		if (variables.getObject(getSipNodeName()) != null) {
-			_logger.warn("Sip node context already defined ", null);
+			_logger.warn("Sip node context already defined ", "");
```

Comment:
from slf4j, warn method can be set without additionnal parameter: `_logger.warn("Sip node context already defined");` may be enough in that case.